implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v1/DynatraceExporterV1.java

```diff
@@ -67,17 +67,22 @@ public class DynatraceExporterV1 extends AbstractDynatraceExporter {
 
     private final NamingConvention namingConvention;
 
+    private static final String AUTHORIZATION_HEADER_KEY = "Authorization";
+
+    private final String authorizationHeaderValue;
+
     public DynatraceExporterV1(DynatraceConfig config, Clock clock, HttpSender httpClient) {
         super(config, clock, httpClient);
 
         this.customMetricEndpointTemplate = config.uri() + "/api/v1/timeseries/";
         this.namingConvention = new DynatraceNamingConvention(NamingConvention.dot, DynatraceApiVersion.V1);
+
+        this.authorizationHeaderValue = String.format("Api-Token %s", config.apiToken());
```

Comment:
Perhaps it is intentional, but by reading `config.apiToken()` only in the constructor, if the config is backed by a dynamic configuration source, updates will not have any effect until a new instance of this is constructed, which I imagine only normally happens on startup of applications. Previously, we would call `config.apiToken()` on each export, allowing the possibility for dynamic updates of the configuration for API token without restarting the application.

Reply:
Ah, you are right. I thought I was very clever by just reading it once, but it changes the behavior, and users might be relying on the original behavior (token rotation while running, etc.). I updated it, so it is read on export again. Thanks for catching this!